client/src/components/PlayingCard/PlayingCard.scss

```diff
@@ -1,19 +1,73 @@
 @import 'styles/variables';
 
 .Card {
-  &__Container {
+  background: #f2efe7;
+  border-radius: 10px;
+  position: relative;
+  box-shadow: 1px 2px 7px rgba($color-black, 0.5);
+  overflow: hidden;
+  width: $base-spacing * 31.25;
+  height: $base-spacing * 43.75;
+
+  &__Stripe {
+    background: $color-black;
+    width: $spacing-large;
     align-items: center;
-    background-color: $color-white;
-    border: $border-large solid $color-primary;
-    border-radius: $radius-regular;
-    box-shadow: $box-shadow;
+    flex-direction: column;
     display: flex;
-    justify-content: center;
-    width: 100%;
-    height: 100%;
+    position: absolute;
+    top: -$spacing-regular;
+    transform: rotate(4deg);
+    bottom: -$spacing-regular;
+    z-index: 1;
```

Comment:
I'm pretty sure there is a z-index problem. See my screenshot in the thread

Reply:
brrblrblrblrlbr